ROI_Maker.py

```diff
@@ -1,115 +1,88 @@
-import SPEFile
-from Updated_Peak_Finder import peak_finder_pro as peak_finder
-import Gamma_Isotopes
-import Gamma_Analysis
-
-def ROI_Maker(spectra, use='NAA'):
+#from Updated_Peak_Finder import peak_finder_pro as peak_finder
+import numpy as np
+import peakutils
 
+def ROI_Maker(spectrum, energy, sub_regions='both'):
```

Comment:
As discussed in my email, ROI_Maker will now decide the subregions based on the peak energy, so no input argument is needed.